usaspending_api/api_contracts/contracts/account/FederalAccountProfile.md

```diff
@@ -4,42 +4,6 @@ HOST: https://api.usaspending.gov
 
 # Federal Account Profile
 
-# Group Landing Page
-
-This endpoint supports the federal account landing page, which provides a list of all federal accounts for which individual federal accounts pages are available on USAspending.gov.
-
-## List Federal Accounts [/api/v2/federal_accounts/]
-
-This endpoint returns a list of federal accounts, their number, name, managing agency, and budgetary resources.
-
-### List Federal Accounts [POST]
-
-+ Request (application/json)
-    + Attributes (object)
-        + `filters` (optional, TimeFilterObject)
-            The filter takes a fiscal year, but if one is not provided, it defaults to the last certified fiscal year.
-        + `sort` (optional, SortObject)
-        + `limit`: 50 (optional, number)
-            The number of results to include per page.
-            + Default: 50
-        + `page`: 1 (optional, number)
-            The page of results to return based on the limit.
-            + Default: 1
-        + `keyword`: test (optional, string)
-            The keyword that you want to search on. Can be used to search by name, number, managing agency, and budgetary resources.
-
-+ Response 200 (application/json)
-    + Attributes
-        + `previous` (optional, number, nullable)
-        + `count`: 1 (required, number)
-        + `limit`: 50 (required, number)
-        + `hasNext`: false (boolean, required)
-        + `page`: 1 (number, required)
-        + `hasPrevious`: false (boolean, required)
-        + `next`: 2 (number, required, nullable)
-        + `fy`: `2018` (string, required)
-        + `results` (array[FederalAccountListing], fixed-type)
-
 # Group Profile Page
```

Comment:
This file still has the `TimeFilterObject`, `SortObject`, and `FederalAccountListing` Data Structures that are no longer used.